other/recon/Check_OPTIONS.bcheck

```diff
@@ -0,0 +1,18 @@
+metadata:
+    language: v1-beta
+    name: "HTTP Methods Available (passive)"
+    description: "Send HTTP OPTIONS method to site to check what methods are avialable via the response"
+    author: "Ceramicskate0"
+    tags: "passive"
+
+given request then
```

Comment:
We think that this would be better as a `given path then` BCheck, to reduce the amount of traffic to the target.

`given path then` will run once for each unique path audited, whereas `given request then` will run for each request audited.

Reply:
Nice ty. Changes made